docs/repository-management.md

```diff
@@ -0,0 +1,134 @@
+# Repository Management
+
+The goal is for this to be a living document explaining how we collaboratively manage the Gutenberg repository. If you’d like to suggest a change, please open an issue for discussion or submit a pull request to the document.
+
+This document covers:
+
+- [Issues](#issues)
+  - [Labels](#labels)
+  - [Milestones](#milestones)
+  - [Triaging Issues](#triaging-issues)
+- [Pull Requests](#pull-requests)
+  - [Code Review](#code-review)
+  - [Design Review](#design-review)
+  - [Merging Pull Requests](#merging-pull-requests)
+  - [Closing Pull Requests](#closing-pull-requests)
+
+## Issues
+
+A healthy issue backlog is one where issues are relevant and actionable. *Relevant* in the sense that they relate to the project’s current priorities. *Actionable* in the sense that it’s clear what action(s) need to be taken to resolve the issue.
+
+Any issues that are irrelevant or not actionable should be closed, because they get in the way of making progress on the project. Imagine the issue backlog as a desk: the more clutter you have on it, the more difficult it is to use the space to get work done.
+
+**Labels**
+
+To better organize the issue backlog, all issues should have [one or more labels](https://github.com/WordPress/gutenberg/labels). Here are some you might commonly see:
+
+
+- [[Type] Bug](https://github.com/WordPress/gutenberg/labels/%5BType%5D%20Bug) - An existing feature is broken in some way.
+- [[Type] Enhancement](https://github.com/WordPress/gutenberg/labels/%5BType%5D%20Enhancement) - Gutenberg would be better with this improvement added.
+- [[Type] Plugin / Extension Conflict](https://github.com/WordPress/gutenberg/labels/%5BType%5D%20Plugin%20%2F%20Extension%20Conflict) - Documentation of a conflict between Gutenberg and a plugin or extension. The plugin author should be informed and provided documentation on how to address.
+- [[Status] Needs More Info](https://github.com/WordPress/gutenberg/labels/%5BStatus%5D%20Needs%20More%20Info) - The issue needs more information in order to be actionable and relevant. Typically this requires follow-up from the original reporter.
+- [Needs Product Decision](https://github.com/WordPress/gutenberg/labels/Needs%20Product%20Decision) - A person with decision-making authority needs to decide whether the project should support this issue.
```

Comment:
Should this be `[Status]`-prefixed?

Reply:
Potentially. In the Dropbox document, we were discussing whether it was a label we wanted to have. If we want to keep it, we can prefix with `[Status]`. Otherwise, we'll need to work out an equivalent.

Reply:
I think we should remove 'Product Decision' until we as a group agree on it's use - we haven't yet. I am fine if we do but just now going through the label, it was being wildly used differently from this. I stand by having a label like this encourages it's overuse and wrong use. For now please remove. Then, let's as a team discuss this and once a decision has been made it can be iterated on by adding back in. We can of course do this if we agree but just a triage of that issue had vastly different interpretations, for now I have removed it's use. I will add back in if we decide - nothing is unable to be reversed.

Reply:
Great clarification, @karmatosed. I've removed the label from the list for now in a4a46f60